### PR TITLE
Add Suport for `google_compute_node_template` to TGC

### PR DESCRIPTION
**Pull request by delimaneto**
adding support for compute.googleapis.com/NodeTemplate
```release-note:none
```

### CODE REVIEW

**Review by sushant00 — APPROVED** [review]

**Comment by sushant00** [comment]
@shuyama1 can you please review the PR now that internal discussion is closed

**Comment by shuyama1** [comment]
@delimaneto would you mind resolving the conflict? Thanks!

File: mmv1/templates/tgc/resource_converters.go.erb [hunk 1]
```diff
@@ -37,23 +37,24 @@ func ResourceConverters() map[string][]cai.ResourceConverter {
 	return map[string][]cai.ResourceConverter{
 		"google_artifact_registry_repository":             {artifactregistry.ResourceConverterArtifactRegistryRepository()},
```
[the remark this answers is not in the document]
Comment (delimaneto) [reply mmv1/templates/tgc/resource_converters.go.erb]: done, thanks (:

**Review by shuyama1 — APPROVED** [review]
Thanks!

**Comment by shuyama1** [comment]
Sorry, forgot to hit the merge button last week

**Pull request merged**